app/components/NewDocumentButton.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { PlusIcon } from "outline-icons";
+import * as React from "react";
+import { useTranslation } from "react-i18next";
+import { Link } from "react-router-dom";
+import Button from "components/Button";
+
+const NewDocumentButton = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Button as={Link} to={"/doc/new"} icon={<PlusIcon />}>
+      {t("New draft doc")}
```

Comment:
```suggestion
      {t("New doc")}
```

It's always been a draft first, no need for the addition here. I do like small reusable components like this, more please!